app/graphql/types/agg_bucket_type.rb

```diff
@@ -1,6 +1,7 @@
 module Types
   class AggBucketType < BaseObject
     field :key, String, null: false
+    field :key_as_string, String, null: true
```

Comment:
We use `key_as_string` for datetime fields because the key provided by Elasticsearch for these range aggs can't be turned into a datetime.